content/get-haiku/r1beta4/_index.md

```diff
@@ -0,0 +1,165 @@
+</pre>
+
+<p>
+	If you are in Haiku, macOS or Linux, you can use the <code>sha256sum</code>
+	command from the Terminal to generate a checksum for your downloaded
+	file and verify against those above. If you are in Windows, you can use
+	a tool such as <a href="https://implbits.com/products/hashtab/" title="Download hashtab">hashtab</a> to display checksums for files directly in explorer.
+</p>
+
+### Cryptographic Signatures
+
+<p>
+To further validate releases, you can also check the cryptographic signature of the release media against our public key using <a href="https://jedisct1.github.io/minisign/">Minisign</a> (available for Haiku, Windows, Linux, BSD, and OS X). The matching .minisig file needs to be placed next to the iso file on your filesystem.
+</p>
+
+<ul>
+  <li><a class="track" href="https://s3.us-east-1.wasabisys.com/haiku-release/r1beta4/haiku-r1beta4-x86_gcc2h-anyboot.iso.minisig">32-bit minisig</a></li>
+  <li><a class="track" href="https://s3.us-east-1.wasabisys.com/haiku-release/r1beta4/haiku-r1beta4-x86_64-anyboot.iso.minisig">64-bit minisig</a></li>
+</ul>
+
+{{< alert-info "Haiku signing key" "Haiku's public signing key is also available at /boot/system/data/trust_db/haiku-2019.pub under recent versions of Haiku." >}}
+
+```shell script
+$ ls
+haiku-r1beta4-x86_64-anyboot.iso
+haiku-r1beta4-x86_64-anyboot.iso.minisig
+
+$ minisign -Vm haiku-r1beta4-x86_64-anyboot.iso -P RWTPfbfFRi5b+T02aw733p5dp/UI/1Z946YWBInmkPmBlx2Zy1Vyw9kP
+Signature and comment signature verified
+Trusted comment: timestamp:1671733559	file:haiku-r1beta4-x86_64-anyboot.iso	hashed
+```
+
+## Upgrading from R1/beta3
+
+There is an upgrade path available for users that are currently using Haiku R1/beta3. If you want to do this upgrade, you will have to use the Terminal (not SoftwareUpdater!) to issue some commmands. To do so,
+
+{{< alert-warning "Warning: only upgrading from Beta 3 is supported!" "It may be possible to upgrade from prior development images, but this is untested and you may run into unknown problems." >}}
+
+The following commands will replace your existing "Haiku" and "HaikuPorts" repositories with their R1/beta3 equivalents:
+
+```shell script
+pkgman add-repo https://eu.hpkg.haiku-os.org/haiku/r1beta4/$(getarch)/current
+pkgman add-repo https://eu.hpkg.haiku-os.org/haikuports/r1beta4/$(getarch)/current
+```
+
+If this is successful, you may then run a `full-sync` to perform the actual upgrade:
+
+```shell script
+pkgman full-sync
+```
+
+After that step has completed successfully, you should immediately reboot the system. You can do this from the Deskbar, or by running ```shutdown -r``` in Terminal.
+
+
+<h3 class="App_People_32">Acknowledgments</h3>
+
+<p style="padding-right:50px;">
+	<p>We thank all the individuals, organizations and companies who generously provide the server space and bandwidth to mirror our releases - your help is very much appreciated!</p>
```

Comment:
Probably should mention all the donations and the Google sponsoring which alnows us to have a full time contractor as well, making this release bigger than usual

Reply:
How should we reference Google's sponsoring? That was just GSoC "stipends", right?

Reply:
We thank all the individuals, organizations and companies who make it possible for the Haiku project to continue its development over the years, including:

- generously providing the server space and bandwidth to mirror our releases
- donating to Haiku inc to allow a developer to work full time on Haiku, pay for our infrastruture hosting costs, and our real-life events like coding sprints,
- or contributing in any other ways to the testing, advertising and development of Haiku.

your help is very much appreciated!